scripts/get-latest-sanctions.ts

```diff
@@ -0,0 +1,53 @@
+import { writeFile, readFile } from "fs/promises";
+import { join } from "path";
+
+import Parser from "rss-parser";
+import { JSDOM } from "jsdom";
+
+const DATA_PATH = join(__dirname, "data/processed-urls.json");
+const OFAC_RSS_URL = "https://ofac.treasury.gov/system/files/126/ofac.xml";
+const parser = new Parser();
+
+(async function main() {
+  const feed = await parser.parseURL(OFAC_RSS_URL);
+
+  const processedURLs: Record<string, string[]> = JSON.parse(
+    (await readFile(DATA_PATH)).toString()
+  );
+
+  for (let { link } of feed.items) {
+    if (!link) continue;
+    if (link in processedURLs) continue;
+
+    try {
+      const body = await fetch(link).then((x) => x.text());
+      const dom = new JSDOM(body);
+      const text =
+        dom.window.document.getElementById("block-ofac-content")?.textContent;
+      if (!text) throw new Error("Couldn't find the right div");
+
+      const RE = /ETH (0x[a-fA-F0-9]{40})/g;
```

Comment:
OFAC will repeat an address for different assets e.g. USDC, ETH, USDT, etc.

It's possible that  ETH isn't mentioned. 

Maybe remove that prefix. Most EVM chains us the 0x prefix with 20bytes so prob can just leave It as '(0x[a-fA-F0-9]{40})/g'